### DISCUSSION TITLE
Is there a way to activate `noscroll` for browser back/forward navigations

### DISCUSSION THREAD

**Discussion by phoenisx** [sveltejs/kit#8385] [Q&A]
I am aware of:
- https://kit.svelte.dev/docs/link-options#data-sveltekit-noscroll and
- `goto`'s `noscroll behaviour.

but even if I disable scroll for these two, when trying to directly navigate back/forward a page using Browser navigation, the page scrolls to the top during navigation.

Since I am using smooth scroll behaviour, the navigation looks pretty uncomfortable and slow.

**Q. Is there a way to disable scroll animation completely for browser back/forward button clicks as well?**

**Comment by phoenisx** [comment]
Okay, I found this https://kit.svelte.dev/docs/modules#$app-navigation-disablescrollhandling 😮‍💨🙇🏽, which does fix my issue but is not a recommended solution.

Is it the only option to disable scrolling behaviour when navigating to a page, using browser navigation buttons?

**Reply by xxfogs** [reply]
New URL: https://svelte.dev/docs/kit/$app-navigation#disableScrollHandling
